src/Uno.UI/Extensions/ViewExtensions.netstd.cs

```diff
@@ -0,0 +1,124 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI
+{
+	public static partial class ViewExtensions
+	{
+		/// <summary>
+		/// Find the first child of a specific type.
+		/// </summary>
+		/// <typeparam name="T">Expected type of the searched child</typeparam>
+		/// <param name="view"></param>
+		/// <param name="childLevelLimit">Defines the max depth, null if not limit (Should never be used)</param>
+		/// <param name="includeCurrent">Indicates if the current view should also be tested or not.</param>
+		/// <returns></returns>
+		public static T FindFirstChild<T>(this UIElement view, int? childLevelLimit = null, bool includeCurrent = true)
+			where T : UIElement
+		{
+			return view.FindFirstChild<T>(null, childLevelLimit, includeCurrent);
+		}
+
+		/// <summary>
+		/// Find the first child of a specific type.
+		/// </summary>
+		/// <typeparam name="T">Expected type of the searched child</typeparam>
+		/// <param name="view"></param>
+		/// <param name="selector">Additional selector for the child</param>
+		/// <param name="childLevelLimit">Defines the max depth, null if not limit (Should never be used)</param>
+		/// <param name="includeCurrent">Indicates if the current view should also be tested or not.</param>
+		/// <returns></returns>
+		public static T FindFirstChild<T>(this UIElement view, Func<T, bool> selector, int? childLevelLimit = null, bool includeCurrent = true)
+			where T : UIElement
+		{
+			Func<UIElement, bool> childSelector;
+			if (selector == null)
+			{
+				childSelector = child => child is T;
+			}
+			else
+			{
+				childSelector = child => child is T t && selector(t);
+			}
+
+			if (includeCurrent && childSelector(view))
+			{
+				return view as T;
+			}
+
+			var maxDepth = childLevelLimit ?? int.MaxValue;
+
+			return (T)view.EnumerateAllChildren(childSelector, maxDepth).FirstOrDefault();
+		}
+
+		public static string ShowDescendants(this UIElement view, StringBuilder sb = null, string spacing = "", UIElement viewOfInterest = null)
+		{
+			sb = sb ?? new StringBuilder();
+			AppendView(view);
+			spacing += "  ";
+			foreach (var child in view._children)
+			{
+				ShowDescendants(child, sb, spacing, viewOfInterest);
+			}
+
+			return sb.ToString();
+
+			StringBuilder AppendView(UIElement innerView)
```

Comment:
Codacy found an issue: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6128211)